src/App.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [sortBy, setSortBy] = useState('all');
+  const [selectTodo, setSelectTodo] = useState<Todo>();
+
+  useEffect(() => {
+    getTodos().then(response => setTodoList(response));
+  }, []);
+
+  const searchFilter = () => {
+    return todoList.filter(todo => todo.title.includes(query));
```

Comment:
We can predict that the user can accidentally press the Caps Lock key, so maybe using 'toLowerCase()' have sense